tests/prepare.sh

```diff
@@ -1,21 +1,23 @@
-#!/bin/bash
-# Prepares a temporary environment to run our tests
+# Prepare a directory with the right testing binaries to avoid conflicts with different versions of floresta.
 #
-# This script should be executed once, before running our functinal test
-# for the first time we are testing a specific commit or in a session, since all the created files are temporary.
+# What this script do ?
 #
-# It'll download and build only utreexod and florestad, since we are not testing the other binaries.
+# 0. While checking build dependencies;
 #
-# Make sure to have python(for the tests), golang(for building utreexod) and rust(for building florestad) installed.
+# 1. Clone and build utreexod at $FLORESTA_TEMP_DIR/binaries/utreexod.
 #
+# 2. Build florestad at $FLORESTA_TEMP_DIR/binaries/florestad.
+#
+# 3. export FLORESTA_TEMP_DIR which points to /tmp/floresta-functional-tests.${HEAD_COMMIT_HASH}/binaries
 
 
-# We expect the current dir is the root dir of the project.
+# We expect for the current dir to be the root dir of the project.
 FLORESTA_PROJ_DIR=$(pwd)
```

Comment:
This could be replaced with `git rev-parse --show-toplevel`. Then you could be inside any subdir and it would know where is the root